app/src/main/java/org/oppia/android/app/player/state/StateViewModel.kt

```diff
@@ -26,7 +26,7 @@ import org.oppia.android.domain.translation.TranslationController
 import org.oppia.android.util.data.AsyncResult
 import org.oppia.android.util.data.DataProviders.Companion.toLiveData
 import org.oppia.android.util.locale.OppiaLocale
-import org.oppia.android.util.platformparameter.EnableLearnerStudyAnalytics
+import org.oppia.android.util.platformparameter.EnableFastInLessonLanguageSwitching
```

Comment:
I keep getting confused when reading this name because "in lesson" isn't hyphenated, so I wonder what "fast in lesson" means.

I wonder if EnableFastLessonLanguageSwitching might be sufficient (or EnableFastLanguageSwitchingInLesson)? In fact you use enableFastLanguageSwitching further down which is somewhat more readable (though I think the "in lesson" part is probably important to keep as part of the name).

Reply:
Seems reasonable. Updated everything to be consistent, and I went with ``EnableFastLanguageSwitchingInLesson`` since it has the same number of characters so I wouldn't have to worry about dealing with any line-wrapping changes.

Reply:
Verified all usages renamed.